src/app/pages/system/update/update.component.html

```diff
@@ -95,33 +95,35 @@
 </mat-card>
 
 <mat-card *ngIf="updates_available && pre_release_train && isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i><strong>This is not a production release, and should only be used for testing.</strong></div>
-  <div class="stable-warning" id="stable_warning_line_2">Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>
+    <strong>{{"This is not a production release, and should only be used for testing." | translate}}</strong>
+  </div>
+  <div class="stable-warning" id="stable_warning_line_2">{{"Before updating, please read the
```

Comment:
I think you'll need to split this into two translated strings:
```html
<div class="stable-warning" id="stable_warning_line_2">
  {{"Before updating, please read the" | translate }}
  <!-- whitespace character -->
  &nbsp;
  <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">{{ "release notes" | translate }}</a>.
</div>
```

Reply:
Splitting single sentence into two parts makes correct translation difficult.  I suppose the problem is in nested {{ }}?  Is there any syntax to express it without splitting?

Reply:
Perhaps the simplest solution would be to make the full sentence an anchor link?
```html
<div class="stable-warning" id="stable_warning_line_2">
  <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">
    {{ "Before updating, please read the release notes." | translate }}
  </a>
</div>
```